cli/tools/bench.rs

```diff
@@ -402,7 +402,7 @@ async fn bench_specifier(
         &specifier.to_file_path().unwrap().display().to_string(),
         false,
       )?;
-      worker.run_event_loop(false).await?;
+      worker.run_event_loop(true).await?;
```

Comment:
Ditto

Reply:
Good catch, but let's do that in a separate PR. I thought I did exclude `--inspect` flags from `deno bench`